deploy/eck-stack/charts/eck-elasticsearch/values.yaml

```
@@ -18,7 +18,7 @@
 
 # Version of Elasticsearch.
 #
-version: 8.10.0-SNAPSHOT
+version: 8.10.2-SNAPSHOT
```

The next version of the Helm eck-stack charts should use the latest version of the Elastic stack which will be released the same day.
```suggestion
version: 8.11.0-SNAPSHOT
```

---

Sorry, I didn't specify but this applies to all eck-stack charts childs:
```
> grep 'version: 8' -r deploy/**/values.yaml
deploy/eck-stack/charts/eck-agent/values.yaml:version: 8.10.2-SNAPSHOT
deploy/eck-stack/charts/eck-beats/values.yaml:version: 8.10.2-SNAPSHOT
deploy/eck-stack/charts/eck-elasticsearch/values.yaml:version: 8.10.2-SNAPSHOT
deploy/eck-stack/charts/eck-fleet-server/values.yaml:version: 8.10.2-SNAPSHOT
deploy/eck-stack/charts/eck-kibana/values.yaml:version: 8.10.2-SNAPSHOT
```